Automattic/wp-calypso
# Signup: Add support for domain only sites

This makes changes to the /domain-first signup flow so that we save a blog option to differentiate domain-only sites from full sites.
  
#### Testing instructions
 
1. Run `git checkout add/domain-only-sites` and start your server, or open a [live branch](https://calypso.live/?branch=add/domain-only-sites)
2. Open the [`Signup` page](http://calypso.localhost:3000/start/domain-first)
3. Go through the signup flow, verifying that you are not offered a free wpcom domain or a domain mapping.
4. When you have finished the flow, look at the response from /me/sites and check whether there is a `is_domain_only` blog option.
 
#### Reviews
